spec.bs

```diff
@@ -1096,6 +1097,14 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
     * To handle an error:
       1. Set |auctionConfig|'s [=auction config/direct from seller signals header ad slot=] to
         failure.
+1. If |config|["{{AuctionAdConfig/deprecatedRenderURLReplacements}}"] [=map/exists=]:
+  1. Let |renderUrlReplacements| be a new empty [=list=].
+  1. [=list/For each=] |replacement| in |config|["{{AuctionAdConfig/deprecatedRenderURLReplacements}}"]:

```

Comment:
{{AuctionAdConfig/deprecatedRenderURLReplacements}} is a promise, right? It's not a list yet.
You need to do something similar to how other promise fields are handled, for example, seller signals: https://github.com/WICG/turtledove/blob/main/spec.bs#L1020

Reply:
Done.